hw/ip/entropy_src/data/entropy_src_sec_cm_testplan.hjson

```diff
@@ -24,70 +24,127 @@
+            Verify the countermeasure(s) CONFIG.REGWEN.
+            Verify that:
+            1) ME_REGWEN and SW_REGUPD cannot be set back to 1 after being set to 0 once.
+            2) If ME_REGWEN is not set, MODULE_ENABLE cannot be modified.
+            3) Only if MODULE_ENABLE is MuBi4False and SW_REGUPD is 1, a) REGWEN reads as 1 and b) associated control and threshold registers can be modified.
+            '''
       stage: V2S
-      tests: []
+      tests: ["entropy_src_rng"]
     }
     {
       name: sec_cm_config_mubi
-      desc: "Verify the countermeasure(s) CONFIG.MUBI."
+      desc: '''
+            Verify the countermeasure(s) CONFIG.MUBI.
+            Verify that upon writing invalid MUBI values to configuration registers:
+            1) the DUT signals a recoverable alert and sets the correct bit in the RECOV_ALERT_STS register, and
+            2) the DUT can be configured back to a safe configuration and the RECOV_ALERT_STS register can be cleared.
+            '''
       stage: V2S
-      tests: []
+      tests: ["entropy_src_rng"]
     }
     {
       name: sec_cm_config_redun
-      desc: "Verify the countermeasure(s) CONFIG.REDUN."
+      desc: '''
+            Verify the countermeasure(s) CONFIG.REDUN.
+            Verify that upon improperly configuring the ALERT_TRESHOLD register:
+            1) the DUT signals a recoverable alert and sets the correct bit in the RECOV_ALERT_STS register, and
+            2) the DUT can be configured back to a safe configuration and the RECOV_ALERT_STS register can be cleared.
+            '''
       stage: V2S
-      tests: []
+      tests: ["entropy_src_rng"]
     }
     {
+      // TODO: It seems we currently always drive either MuBi8True or MuBi8False values to these signals.
+      // Similar to CONFIG.MUBI and CONFIG.REDUN we should probably also with a certain probability drive invalid MuBi values to these signals.
+      // The DUT isn't expected to signal an alert though in case of invalid encodings for these two signals.
       name: sec_cm_intersig_mubi
-      desc: "Verify the countermeasure(s) INTERSIG.MUBI."
+      desc: '''
+            Verify the countermeasure(s) INTERSIG.MUBI.
+            Verify that unless the otp_en_entropy_src_fw_read or otp_en_entropy_src_fw_over input signals are equal to MuBi8True the DUT doesn't allow reading entropy from the ENTROPY_DATA register or from the FW_OV_RD_DATA register, respectively.
+            '''
       stage: V2S
-      tests: []
+      tests: ["entropy_src_rng"]
     }
     {
       name: sec_cm_main_sm_fsm_sparse
-      desc: "Verify the countermeasure(s) MAIN_SM.FSM.SPARSE."
+      desc: '''
+            Verify the countermeasure(s) MAIN_SM.FSM.SPARSE.
+            The entropy_src_functional_errors test verifies that if the FSM state is forced to an illegal state encoding this is reported in the ERR_CODE register.
+            It seems that it doesn't check whether the DUT actually triggers a fatal alert.
```

Comment:
Ideally, we would adapt the test but since time is very very short, let's leave it as is. The formal tests verify the alert connection and triggering. That's sufficient.